pkg/storage/utils/decomposedfs/spaces.go

```diff
@@ -387,16 +387,32 @@ func (fs *Decomposedfs) UpdateStorageSpace(ctx context.Context, req *provider.Up
 	}
 	space.Owner = u
 
+	space.SpaceType = filepath.Base(filepath.Dir(matches[0]))
+
+	metadata := make(map[string]string)
```

Comment:
Nitpick: Shouldn't you add the size to the make here? It is static and known... I think that is more efficient.

Reply:
Yes. Of course. The thing is, we don't know how many props the Client will set in the request, none of them is mandatory.